projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="mkbootimg"
+PKG_VERSION="c2bfa15"
```

Comment:
Full rev `c2bfa15aa5bfc4e3f1b32de28e327b80b37db1cf` (sha256: `c980a0c59eed3635b5c9761781b12047256b48fa0a2e6f3ce639c5db4629290a`)?

This is also available with tag `1.6.0` (sha256: `7930b7dba25e5b4a39e6d19ccce9005e17c98a2b6c9e1aa6408a88c716755aaf`)